src/qt/listctrl.cpp

```diff
@@ -1836,14 +1836,33 @@ void wxListCtrl::CheckItem(long item, bool check)
     m_model->CheckItem(item, check);
 }
 
-void wxListCtrl::SetSingleStyle(long WXUNUSED(style), bool WXUNUSED(add))
+void wxListCtrl::SetSingleStyle(long style, bool add)
 {
+    long flag = GetWindowStyleFlag();
+
+    // Get rid of conflicting styles
+    if (add)
+    {
+        if (style & wxLC_MASK_TYPE)
+            flag = flag & ~wxLC_MASK_TYPE;
```

Comment:
Very minor, but personally I find
```suggestion
            flag &= ~wxLC_MASK_TYPE;
```
more readable.

Reply:
It was copied from wxMSW implementation basically.

Reply:
Updated.